scripts/verify-promote

```diff
@@ -104,11 +106,17 @@ banner "Verify tarball install and that 'sf update $NEW_CHANNEL' version matches
 # If you are testing this locally, you'll need to use a tarball built for your OS
 #   There is a VERSIONS_URL for M1 macs above that you can uncomment
 #   Other URLs can be found here: https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_install_cli.htm#sfdx_setup_install_cli_olderversions
-OLDER_VERSION_URL=$(echo "$VERSION_CONTENT" | jq 'to_entries[7]' | jq -r '.value')
+OLDER_VERSION_URL=$(echo "$VERSION_CONTENT" | jq 'to_entries[15]' | jq -r '.value')
```

Comment:
This change is to grab a version older than `stable`. With `nightly` builds we were grabbing a newer version at index `7`